bobs_shop.php

```diff
@@ -0,0 +1,149 @@
+<body style="background-color: powderblue">
+    <nav class="navbar navbar-dark bg-primary navbar-collapse" style="border-bottom: 5px solid black">
+        <a class="display-4 font-weight-bold"><b> Bob's Emporium </b></a>
+        <a class="n-item navbar-text text-white display-4 font-weight-bold">Home</a>
+        <a class="n-item navbar-text text-white display-4 font-weight-bold">About</a>
+    </nav>
+    <h1 class="d-flex justify-content-center align-items-center container mt-4 font-weight-bold">Contact Us!</h1>
+    <span class="Error d-flex justify-content-center align-items-center container" id="Error" ></span>
+    <div class="d-flex justify-content-center align-items-center container mt-1">
+        <form id="contactForm" class="card text-white bg-primary mb-3 card border-dark" name="formd" style="max-width: 30rem;">
+            <div class="align-items-center card-body">
+                <div class="row">
+                    <div class="col">
+                        <label for="First">First Name</label>
+                        <input type="text" name="First" id="First" placeholder="First" class="form-control">
+                    </div>
+                    <span class="col">
+                        <label for="Last">Last Name</label>
+                        <input type="text" name="Last" id="Last" placeholder="Last" class="form-control">
+                    </div>
+                </div>
+                <div class="col-auto">
+                    <label for="Email">Email</label>
+                    <input type="email" name="Email" id="Email" placeholder="E-Mail Address" class="form-control form-control-lg">
+                </div>
+                <div class="col">
+                    <label for="Message">Message</label>
+                    <textarea type="text" id="Message" name="Message" placeholder="Send Us A Quick Message!" class="form-control" style="height: 25vh;"></textarea>
+                </div>
+                <input type="hidden" id="g-recaptcha-response" name="g-recaptcha-response"/>
+                <input type="hidden" name="dateTime"/>
+                <div class="row">
+                    <div class="col">
+                        <button class="btn btn-secondary btn-lg btn-block mt-3 mb-1 w-75 ml-5 form-control" type="submit" name="Submit" id="submitButton"><b>Send</b></button>
+                    </div>
+                </div>
+            </form>
+        </div>
+</body>
+</html>
+<script>
+grecaptcha.ready(function() {
+        grecaptcha.execute('<?php echo SITE_KEY; ?>', {action: '/'}).then(function(token) {
+            document.getElementById("g-recaptcha-response").value = token;
+        });
+    });
+
+    var Form = document.getElementById("contactForm");
+    
+    var error = document.getElementById("Error");
+
+    function Reset(e){
+        error.innerHTML = null;
+        Form.Submit.innerHTML = "<b>Send</b>";
+        e.target.setAttribute("class","form-control")
+    }
+    
```

Comment:
If I were able to do an ajax API call the function would have been as follows:
"
function Do(e){
        e.preventDefault()
        let key = "6Ldia70UAAAAAFKhdZsL1eqcpRZ4k4ih0Vz29eg7";
        let token = document.getElementById("g-recaptcha-response").value;
        var url = "https://www.google.com/recaptcha/api/siteverify?secret="+key+"&response="+token;
        //console.log("Token: " + token)
        var request = $.ajax({
        url:url,
        type:"POST",
        contentType:"application/json; charset=utf-8",
        dataType:"json",
        success: function(response, textStatus){
            Submit();
        },
        error: function(req, response){
            console.log("I Failed Master, Error: " + response)
        }
    });
    }
"